venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -165,26 +189,104 @@ function() {
+            },
+            writers: {
+              'values-regex': '~.*|\\(anonymous\\)'
+            },
+            content:{
+              title: {
+                order: 0,
+                'value-regex': '.{1,500}',
+                description: 'Brief summary of your comment.',
+                required: true
+              },
+              comment: {
+                order: 1,
+                'value-regex': '[\\S\\s]{1,5000}',
+                description: 'Your comment or reply.',
+                required: true
+              }
+            }
+          }
+        }
+
+        var officialCommentInvitation = {
+          id: CONF + '/-/Paper' + savedNote.number + '/Official_Comment',
+          signatures: [CONF],
+          writers: [CONF],
+          invitees: [reviewerGroupId, authorGroupId, areachairGroupId, PROGRAM_CHAIRS],
+          readers: ['everyone'],
+          reply: {
+            forum: savedNote.id,
+            replyto: null,
+            readers: {
+              description: 'The users who will be allowed to read the above content.',
+              'value-dropdown': ['everyone', REVIEWERS_PLUS, AREA_CHAIRS_PLUS, PROGRAM_CHAIRS]
+            },
+            signatures: {
+              description: 'How your identity will be displayed with the above content.',
+              'values-regex': [reviewerGroupId, authorGroupId, areachairGroupId, PROGRAM_CHAIRS].join('|')
+            },
+            writers: {
+              'values-regex': [reviewerGroupId, authorGroupId, areachairGroupId, PROGRAM_CHAIRS].join('|')
+            },
+            content:{
+              title: {
+                order: 0,
+                'value-regex': '.{1,500}',
+                description: 'Brief summary of your comment.',
+                required: true
+              },
+              comment: {
+                order: 1,
+                'value-regex': '[\\S\\s]{1,5000}',
+                description: 'Your comment or reply.',
+                required: true
+              }
+            }
+          }
+        }
+
         var groupPromises = Promise.all([
           or3client.or3request(or3client.grpUrl, authorGroup, 'POST', token),
-          or3client.or3request(or3client.inviteUrl, withdrawPaperInvitation, 'POST', token),
           or3client.or3request(or3client.grpUrl, reviewerGroup, 'POST', token),
           or3client.or3request(or3client.grpUrl, areachairGroup, 'POST', token),
           or3client.or3request(or3client.grpUrl, anonReviewer1Group, 'POST', token),
           or3client.or3request(or3client.grpUrl, anonReviewer2Group, 'POST', token),
           or3client.or3request(or3client.grpUrl, anonReviewer3Group, 'POST', token),
           or3client.or3request(or3client.grpUrl, anonReviewer4Group, 'POST', token),
           or3client.or3request(or3client.grpUrl, anonReviewer5Group, 'POST', token),
+          or3client.or3request(or3client.inviteUrl, withdrawPaperInvitation, 'POST', token),
```

Comment:
I realized that we can get the 'deleted' notes if that is the issue of displaying the withdrawn paper in the ICLR home page. We wouldn't need this invitation and we can user the current deletion functionality.

Reply:
ok, that makes sense. My only worry with this solution is that it may be confusing for the user. They may expect the "delete" functionality to delete it and hide it. As opposed to "withdraw", which doesn't necessarily imply that the note will be hidden